example/90-shoot.yaml

```diff
@@ -147,6 +147,15 @@ spec:
   # workersSettings:
   #   sshAccess:
   #     enabled: false
+# etcd:
+#   main:
+#     autoscaling:
+#       cpu: "2"
+#       memory: 6Gi
```

Comment:
In the issue, we agreed on putting those under `minAllowed`. This allows to further augment the `autoScaling` API section in case we want to add more configurations knobs in the future.

Update: It seems the Golang API definition has `minAllowed`, it's just missing from the YAML examples.